docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -420,6 +422,18 @@ table description supplier are:
 * There is no need to define tables manually.
 * Some Protobuf specific types like ``oneof`` are supported and mapped to JSON.
 
+When using Protobuf decoder with the Confluent table description supplier, the
+following additional steps must be taken:
+
+- Copy the JARs ``kafka-protobuf-provider`` and ``kafka-protobuf-types`` for
```

Comment:
Where do you get those from?

Reply:
https://packages.confluent.io/maven/io/confluent/